clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/DataRecoveryClient.java

```diff
@@ -42,6 +48,21 @@ public void execute(DataRecoveryParams drParams, StoreRepushCommand.Params cmdPa
     getExecutor().shutdownAndAwaitTermination();
   }
 
+  public Integer estimateRecoveryTime(DataRecoveryParams drParams, EstimateDataRecoveryTimeCommand.Params cmdParams) {
+    Set<String> storeNames = drParams.getRecoveryStores();
+    if (storeNames == null || storeNames.isEmpty()) {
+      LOGGER.warn("store list is empty, exit.");
+      return -1;
```

Comment:
If there is a empty list, maybe we could simply return 0. Recovering 0 store takes 0 seconds and we have print out the warning message.

Reply:
Adjusted the logic here along with using the Result data type :)